### PR TITLE
Fix typo (a --> an, an --> a)

### PR DESCRIPTION
**Pull request by zerotypos-found**

### CODE REVIEW

File: docs/Generics.rst [hunk 1]
```diff
@@ -594,7 +594,7 @@ lets us describe an iteration of values of some given value type::
     func next() -> Element
   }
 
-Now, we want to express the notion of an enumerable collection, which provides a
+Now, we want to express the notion of an enumerable collection, which provides an
```
Comment (gribozavr) [inline docs/Generics.rst]: I don't think we need 'an' at all here.

Comment (zerotypos-found) [inline docs/Generics.rst]: Do you mean
&nbsp;&nbsp;  which provides a iteration --> which provides iteration
?

Comment (gribozavr) [inline docs/Generics.rst]: Yes.

Comment (zerotypos-found) [inline docs/Generics.rst]: OK, done.

**Comment by gottesmm** [comment]
Can you squash the commits and do a force push to the branch?

**Comment by zerotypos-found** [comment]
Sure, done.

**Comment by lattner** [comment]
Thanks!

**Pull request merged**